rldb.py

```diff
@@ -112,18 +112,18 @@ def end_creation(user, channel, message_id):
 
 
 def exists(message_id):
-    db.execute(f"SELECT * FROM messages WHERE message_id = '{message_id}';")
+    db.execute("SELECT * FROM messages WHERE message_id = ?;", (message_id,))
```

Comment:
Do the commas after, for example, `message_id` serve a purpose or can I delete them? I do not see examples in the sqlite3 library using trailing commas.

Reply:
Yes, it's for forcing tuples